ibm_mq/metadata.csv

```diff
@@ -1,5 +1,6 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
 ibm_mq.channel.channels,gauge,,resource,,The number of active channels,0,ibm_mq,active channel count
+ibm_mq.channel.count,gauge,,,,Sum by status to count channels. Filter by channel and status tags to create notifications.,0,ibm_mq,channel count
```

Comment:
Is it a sum? status_active is either 0 or 1, right?

Reply:
Yes, it's either 0 or 1, then you can get status by summing by status.

For example, you can get the "total for all channels in the status running".